spec/layout_register_test.spec.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+describe(\Dxw\Iguana\Theme\LayoutRegister::class, function () {
+	beforeEach(function () {
```

Comment:
I don't think this stuff with the template file should be in the setup/teardown, it's a separate test